### PR TITLE
feat: tabs for teams in mobile 

### LINKED ISSUE #6501: [CAL-798] mobile: /event-types - tabs for teams
* [ ] fix margin
* [ ] add tabs for teams (if available. if no teams, hide tabs)

bad, too much margin:

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-798](https://linear.app/calcom/issue/CAL-798/mobile-event-types-tabs-for-teams)</sub>

### PR DESCRIPTION
**Pull request by G3root**
## What does this PR do?

Fixes #6501 

## **changes to the core components:**

- [x] added a avatar prop for `HorizontalTabItem` component
- [x] fixed background color of selected  `HorizontalTabItem` component.
- [x] fixed padding left and right of `HorizontalTabItem` component.

before:
Screen Shot 2023-01-25 at 19 40 53(1)

after:
Screen Shot 2023-01-25 at 19 41 57(1)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by G3root** [comment]
@Jaibles can you check is everything looks good in the preview deployment

**Comment by Jaibles** [comment]
@G3root looks great 👏 Can you make two tiny changes for me please...
https://user-images.githubusercontent.com/4536123/214561093-8b3b98e4-8c87-48ff-97b0-f234726f1914.mp4

**Comment by G3root** [comment]
@Jaibles pushed those changes

File: apps/web/pages/event-types/index.tsx [hunk 1]
```diff
@@ -74,6 +78,40 @@ interface EventTypeListProps {
...
+    <div>
+      <HorizontalTabs tabs={tabs} />
+      {events.length ? (
+        <EventTypeList
+          types={events[0].eventTypes}
+          group={events[0]}
+          groupIndex={0}
+          readOnly={events[0].metadata.readOnly}
+        />
+      ) : null}
```
Comment (sean-brydon) [inline apps/web/pages/event-types/index.tsx:110]:
```suggestion
      {events.length && (
        <EventTypeList
          types={events[0].eventTypes}
          group={events[0]}
          groupIndex={0}
          readOnly={events[0].metadata.readOnly}
        />
      )}
```

File: packages/ui/components/navigation/tabs/HorizontalTabItem.tsx [hunk 2]
```diff
@@ -45,7 +48,13 @@ const HorizontalTabItem = function ({ name, href, linkProps, ...props }: Horizon
           aria-hidden="true"
         />
       )}
-      {isLocaleReady ? t(name) : <SkeletonText className="h-4 w-24" />}
+      {isLocaleReady ? (
+        <div className="flex items-center gap-x-2">
+          {avatar ? <Avatar size="sm" imageSrc={avatar} alt="avatar" /> : null} {t(name)}
```
Comment (sean-brydon) [inline packages/ui/components/navigation/tabs/HorizontalTabItem.tsx:53]:
```suggestion
          {avatar && <Avatar size="sm" imageSrc={avatar} alt="avatar" />} {t(name)}
```

**Review by sean-brydon — APPROVED** [review]
Nice job :) Works perfectly locally

**Pull request merged**